bindings/bind_py_structure_manager.cc

```diff
@@ -360,23 +360,24 @@ struct BindAdaptor<AdaptorNeighbourList, Implementation_t> {
   using ImplementationPtr_t = std::shared_ptr<Implementation_t>;
   using PyManager_t = PyManager<Manager_t>;
   static void bind_adaptor_init(PyManager_t & adaptor) {
-    adaptor.def(py::init<std::shared_ptr<Implementation_t>, double, bool>(),
-                py::arg("manager"), py::arg("cutoff"),
-                py::arg("consider_ghost_neighbours") = false,
-                py::keep_alive<1, 2>());
+    adaptor.def(
+        py::init<std::shared_ptr<Implementation_t>, double, bool, double>(),
+        py::arg("manager"), py::arg("cutoff"),
+        py::arg("consider_ghost_neighbours") = false, py::arg("skin") = 0.,
+        py::keep_alive<1, 2>());
   }
 
   static void bind_adapted_manager_maker(const std::string & name,
                                          py::module & m_adaptor) {
     m_adaptor.def(
         name.c_str(),
         [](ImplementationPtr_t manager, const double & cutoff,
-           const bool & consider_ghost_neighbours) {
+           const bool & consider_ghost_neighbours, const double & skin) {
```

Comment:
So that we can nudge atoms by small amounts (e.g. to test gradients) without having to recompute the neighbourlist. This will be relevant especially for MD simulations later on.

Reply:
skin is the parameter of the Verlet list kind of rational. This is one of the new feature of the code:
updating the manager with a similar structure might not trigger an update of the linked cell partitioning depending on the skin parameter. Note the rest of the adaptors will be updated at the moment

Reply:
a possible improvement is to have a similar rational in adaptor strict so that only the distances get updated and not the whole NL

Reply:
Closing because this is really the subject of a separate PR